tests/unit/fake_data_root/var/log/rabbitmq/rabbit@juju-ff6a31-19-lxd-18.log

```diff
@@ -1,2 +1,7 @@
+
 operation queue.declare caused a channel exception not_found: failed to perform operation on queue 'test_exchange_queue' in vhost 'nagios-rabbitmq-server-0' due to timeout
 
+Mirrored queue 'rmq-two-queue' in vhost '/': Stopping all nodes on master shutdown since no synchronised slave is available
+
+Discarding message {'$gen_call',{<0.753.0>,#Ref<0.989368845.173015041.56949>},{info,[name,pid,slave_pids,synchronised_slave_pids]}} from <0.753.0> to <0.943.0> in an old incarnation (3) of this node (1)
```

Comment:
Is there a way to add a second entry in `bugs.yaml` for the same issue, or use 2 `expr`?

Reply:
yes, the expr field can be a list as well as a string - see https://github.com/canonical/hotsos/blob/6a0d650a8d76b5a5f85f4ddc8c0a9f8939e1de7a/defs/bugs.yaml#L17